api/email/send_email.py

```diff
@@ -0,0 +1,47 @@
+import smtplib
+import time
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+from SoftwareUtilities.logging.logger import Logger
+
+
+class SendEmail:
+    _host = 'smtp.office365.com'
+    _port = 587
+
+    def __init__(self, user_name, password):
+        self._user_name = user_name
+        self._password = password
+
+    def init_connection(self):
```

Comment:
set this function as protected
```suggestion
    def _init_connection(self):
```